source/lmp/compute_deeptensor_atom.cpp

```diff
@@ -155,7 +158,7 @@ void ComputeDeeptensorAtom::compute_peratom() {
     // record when selected and in group
     if (selected && ingroup) {
       for (int jj = 0; jj < size_peratom_cols; ++jj) {
-        tensor[ii][jj] = atensor[iter_tensor + jj];
+        tensor[ii][jj] = atensor[iter_tensor + jj] * dist_unit_cvt_factor;
```

Comment:
@Yi-FanLi and @wanghan-iapcm, I have a question here. Do we expect the output all tensorial models to have the unit of distance?

Reply:
Generally no. But at least in my knowledge and as introduced in the documentation, the usage of the compute deeptensor/atom feature is restricted to predicting the displacements of Wannier centroids, which do have the unit of distance. I suggest keeping the distance unit here at this moment, and see if there will be other ways of using compute deeptensor/atom. If there are, we should also add them in the document later on.

Reply:
In the current documentation, I don't see it restricted to predicting the displacements of Wannier centroids.

![image](https://github.com/deepmodeling/deepmd-kit/assets/9496702/61c928c3-3e63-4153-ac45-39d1cb6c97de)

Reply:
In principle, the tensor can be any property. However, in practice, dipole (as shown in the example of the screenshot) is the only case I have ever seen. The practical use of this feature does not exceed the case of the dipole example. This, assuming that the tensor has the unit of displacement should be a reasonable assumption. I realize that the tensor can also be polarizability, as shown in the original paper of Deep Wannier. However, I did not see users actually using it in practice. In the future, if some users need to use non-metal units for different quantities, we can add an interface to type the name of the quantity so that we can determine the conversion factors accordingly. Currently, only supporting displacement should be enough.

Reply:
This behavior should be documented, since in the documentation, we say that the unit of the tensors can be any.

Reply:
The features added in this PR have been discussed in detail in the document.